agent/containers/images/Makefile

```diff
@@ -127,11 +102,11 @@ tm: pkgmgr-clean all-tags $(_DEFAULT_DISTROS:%=%-tool-meister-tagged)
 #-
 
 # We also offer targets per distribution target
-${_DISTROS}: %: pkgmgr-clean %-all-tagged
+${_DISTROS}: %: %-all-tagged
 
-$(_DISTROS:%=%-tds): %-tds: pkgmgr-clean %-tool-data-sink-tagged
+$(_DISTROS:%=%-tds): %-tds: %-tool-data-sink-tagged
 
-$(_DISTROS:%=%-tm): %-tm: pkgmgr-clean %-tool-meister-tagged
+$(_DISTROS:%=%-tm): %-tm: %-tool-meister-tagged
```

Comment:
Is there some reason why these targets should _not_ depend on the `pkgmgr-clean` target?

(That dependency was replaced with `all-tags` which in turn depends on `pkgmgr-clean`, but now that we're reverting that change, shouldn't we be restoring the original dependency?)

Reply:
This was changed from `all-tags` to `pkgmgr-clean` in PR #2607 with no particular comment that I could find describing the change.  We have `pkgmgr-clean` as a dependency of `all-tags` to help ensure all the distributions are using the same pbench-agent version.  Why we need this per distribution, where the version for that distribution is self-consistent, and not each time we generate a `*-tags.lis` file is still unclear.  I had removed it because of that inconsistency.

I'll replace it in order for this PR to move forward and somebody else can solve that problem in a future PR.

Reply:
The `all-tags` dependency was removed because it prevented these targets from building when there was a problem with any of the default distros on COPR.  (E.g., we couldn't build Fedora-36 unless we had a RHEL-7 RPM.)

Previously, these targets depended on `pkgmgr-clean` indirectly via their dependency on `all-tags`; so, when I removed the `all-tags` dependency, I replaced it with `pkgmgr-clean` to maintain that dependency.

I still don't know why we need to clean the dnf cache before building the `*-tags.lis` files, but it seems like, if we need to do it before doing a multiple-distro build, then we should need to do it before doing a single-distro build...but maybe I'm just confused.